### DISCUSSION TITLE
OpenCL support, TESSERACT_OPENCL_DEVICE

### DISCUSSION THREAD

**Discussion by yepack** [hertzg/tesseract-server#99] [Q&A]
Is this server support OpenCL GPU or using just native CPU performance?

**Comment by yepack** [comment]
https://tesseract-ocr.github.io/tessdoc/TesseractOpenCL.html
```

Known problems

OpenCL is still experimental, so expect possible crashes, wrong OCR results or bad instead of improved performance.
```

**Comment by hertzg** [comment]
This project does not build the tesseract itself, it uses the one provided by the base docker image, Alpine.

You can search for the compile flags, build log and the overall APKBUILD file on the respective links from the package defition.

https://pkgs.alpinelinux.org/packages?name=tesseract*&branch=v3.15&repo=&arch=&maintainer=
